draft/spec/index.html

```diff
@@ -854,7 +854,17 @@ <h2>Logs Directory</h2>
                 wrong, while others may wish to only store a log entry if an intervention was required.
             </p>
         </blockquote>
+    </section>
 
+    <section id="extensions">
+        <h2>Extensions directory</h2>
+        <p>
+            The base directory of an OCFL Object MAY contain a directory named <code>extensions</code> for the purposes
+            of extending the functionality of an OCFL Object. Sub-directories of the extensions directory MUST be named
+            according to a registered extension name. OCFL Validators MUST ignore any content in extensions that are
+            not understood. The specific structure and function of the extension, as well as a declaration of the
```

Comment:
> not understood

that's a little unclear.  we should be a bit more specific.  i'm assuming you mean by a validator that understands the spec and/or that can validate whatever the extension asks it to validate.

Reply:
I took the language from the existing §4.4, "An OCFL validator must ignore any files in the storage root it does not understand."

We also have (in §3.4): "Optional fixity algorithms that are not supported by a client must be ignored by that client."

So would something like "Extension directories that are not supported by a client MUST be ignored by that client" make sense? Should we narrow it down specifically to validation, or just leave it open generally to "clients"?

Reply:
I think it should be "clients" and agree with last wording suggestion

Reply:
I think stick with validators - this is the spec for objects at rest. We can discuss clients in the Implementation Notes (Client Behaviour's section) - ignoring extentions when writing is dangerous.